api/cluster/pdb_test.go

```diff
@@ -17,13 +17,18 @@ func TestPodDisruptionBudget_BuildPDBSpec(t *testing.T) {
 	defaultInt := 20
 	defaultIntOrString := intstr.FromString("20%")
 	defaultLabels := map[string]string{
+		"gojek.com/app":    "sklearn-sample-s",
+		"model-version-id": "1",
+	}
+	defaultSelectors := map[string]string{
 		"gojek.com/app": "sklearn-sample-s",
 	}
 
 	type fields struct {
 		Name                     string
 		Namespace                string
 		Labels                   map[string]string
+		Selectors                map[string]string
```

Comment:
can you add new test for function `NewPodDisruptionBudget`

Reply:
Sure, I've added the unit test for this